lib/chef/azure/commands/enable.rb

```diff
@@ -66,9 +66,9 @@ def enable_chef
     begin
       configure_chef_only_once
 
-      install_chef_service if @exit_code == 0
+      install_chef_service if @exit_code == 0 && windows?

```

Comment:
Perhaps check `is_started?` on the service and invoke install and enable if its not to avoid platform specific code.